### DISCUSSION TITLE
Css parsed falesly in native css mode

### DISCUSSION THREAD

**Discussion by janlent1** [webpack/webpack#16862] [Q&A]
# Bug report

**What is the current behavior?**
```
.item div:not(.disabled, .mButtonDisabled, .tipOnly) {
    p-events: initial !important;
}
```
gets transpiled to
```
.item div:not(.disabled, .mButtonDisabled, .tipOnly {
    p-events: initial !important;
}
```
The closing parenthesis are missing.

**If the current behavior is a bug, please provide the steps to reproduce.**
Set up a small repo with css modules enabled and add this to a example css file. This should then be output to something like this. 

**What is the expected behavior?**
```
.item div:not(.disabled, .mButtonDisabled, .tipOnly) {
    p-events: initial !important;
}
```
should be  transpiled to
```
.item div:not(.disabled, .mButtonDisabled, .tipOnly) {
    p-events: initial !important;
}
```

**Other relevant information:**
webpack version:  5.76.2
Node.js version: 14.21.1
Operating System: Linux
Additional tools:

**Comment by alexander-akait** [comment]
Please provide reproducible test repo

**Reply by janlent1** [reply]
Will create one right away!

**Reply by janlent1** [reply]
@alexander-akait https://github.com/janlent1/nativecssIssue npm install && npm run build 
Then you'll see the parsed css is faulty.

**Reply by janlent1** [reply]
seems like left parenthesis processing is skipped for some reason. I dont understand how the right parenthesis is then removed in the end

**Reply by janlent1** [reply]
seems like here consumePotentialPseudo we get to the left parenthesis and they are just skipped
if (cc === CC_LEFT_PARENTHESIS) {
		pos++;
                ^

**Reply by janlent1** [reply]
https://github.com/webpack/webpack/pull/16864 this seems to work for this code.
Lets see whether that could be the fix.

**Reply by janlent1** [reply]
seems like not(.item1, .item2) with multiple items inside the parentheses are not regarded in the parsing logic, because the comma breaks the mode logic. The mode is getting removed and then the closing parenthesis expects there to be a value other than undefined. But there isn't.
